bootloader/Makefile

```diff
@@ -15,3 +16,6 @@ grub2-installer/icons/%.png: grub2-installer/icons/%.svg
 grub2-installer/dejavu.pf2:
 	grub-mkfont -s 20 "$(dejavu)/share/fonts/truetype/DejaVuSans.ttf" -o $@
 
+install: grub2-installer-theme.tar.gz
+	mkdir -p "$(install_dest)"
+	tar --strip-components=1 --directory "$(install_dest)" -xf "$<"
```

Comment:
This is silly, but eh, this tests the tar.gz build via the Nix output. If/when CI/CD is done in here, this ensures the tarball is "tested".